examples/simulation/plot_simulated_sample_dataset.py

```diff
@@ -0,0 +1,181 @@
+# of the 4 areas.
+
+region_id = 0
+source_simulator = mne.simulation.SourceSimulator(src, tstep=tstep)
+for i, region_name in enumerate(region_names):
+    region_id = i+1
+    events_tmp = events[np.where(events[:, 2] == region_id)[0], :]
+    for j in range(2):
+        label_name = activations[region_name][j][0]
+        label_tmp = mne.read_labels_from_annot(subject, annot,
+                                               subjects_dir=subjects_dir,
+                                               regexp=label_name,verbose=False)
+        label_tmp = label_tmp[0]
+        amplitude_tmp = activations[region_name][j][1]
+        source_simulator.add_data(label_tmp, amplitude_tmp*source_time_series,
+                                  events_tmp)
+
+# To obtain a SourceEstimate object, we need to use `get_stc()` method of
+# SourceSimulator class.
+stc_data = source_simulator.get_stc()
+
+
+# Simulate raw data.
+
+# Project the source time series to sensor space. Three types of noise will be
+# added to the simulated raw data:
+#     - multivariate Gaussian noise computed using a noise covariance from
+#       original epoched data
+#     - blink (EOG) noise
+#     - ECG noise
+# The source simulator can be given directly to the `simulate_raw` function.
+
+raw_sim = mne.simulation.simulate_raw(info, source_simulator, forward=fwd,
+                                      cov=None)
+noise_epochs = mne.Epochs(raw, events, tmax=0)
+noise_epochs.info['bads'] = []
+noise_cov = mne.compute_covariance(noise_epochs)
+
+mne.simulation.add_noise(raw_sim, cov=noise_cov, random_state=0)
+mne.simulation.add_eog(raw_sim, random_state=0)
+mne.simulation.add_ecg(raw_sim, random_state=0)
+
+# Plot original and simulated raw data.
+raw.plot()
+raw_sim.plot(title='Simulated raw data from sample dataset')
+
+################################################################################
+# Reconstruct simulated source time courses using dSPM inverse operator
+#----------------------------------------------------------------------
+
+# Here, source time courses for auditory and visual areas are reconstructed
+# separately and their difference is shown. This was done merely for better
+# visual representation of source recontruction.
+# As expected, when high activations appear in primary auditory areas, primary
+# visual areas will have low activations and vice versa.
+method, lambda2 = 'dSPM', 1. / 9.
+epochs = mne.Epochs(raw_sim, events, event_id)
+inv = mne.minimum_norm.make_inverse_operator(epochs.info, fwd, noise_cov)
+stc_aud = mne.minimum_norm.apply_inverse(
+    epochs['auditory/left'].average(), inv, lambda2, method)
+stc_vis = mne.minimum_norm.apply_inverse(
+    epochs['visual/right'].average(), inv, lambda2, method)
+stc_diff = stc_aud - stc_vis
+
+brain = stc_diff.plot(subjects_dir=subjects_dir, initial_time=0.1, hemi='split', 
+                     title='Difference between reconstructed stcs of auditory'
```

Comment:
note that the result of the `title` parameter is not visible in the tutorials, it only sets the title of the plot window that pops up during interactive python sessions.

Reply:
Is it ok to leave it anyways?